migration/20200702-add-licensepool-self_hosted-field.sql

```diff
@@ -1 +1,11 @@
-ALTER TABLE licensepools ADD COLUMN self_hosted BOOLEAN;
\ No newline at end of file
+DO $$ 
+ BEGIN
+  -- Add the 'self_hosted' column
+  BEGIN
+   ALTER TABLE licensepools ADD COLUMN self_hosted BOOLEAN;
+  EXCEPTION
+   WHEN duplicate_column THEN RAISE NOTICE 'column licensepools.self_hosted already exists, not creating it.';
+  END;
+ END;
+$$;
+
```

Comment:
This is an unrelated change. While writing the migration script I noticed that the most recent other migration script didn't have the safety code we use to make sure the migration process doesn't crash if given a database that was already partially migrated.